spec/mailers/scheduled_publish_mailer_spec.rb

```diff
@@ -1,75 +1,133 @@
 # frozen_string_literal: true
 
 RSpec.describe ScheduledPublishMailer do
-  before do
-    @user = create(:user)
+  let(:recipient) { build(:user) }
 
-    published_at = Time.current
-    @publish_time = published_at.strftime("%-l:%M%P")
-    @publish_date = published_at.strftime("%d %B %Y")
-  end
+  describe ".success_email" do
+    it "creates an email for the user" do
+      edition = build(:edition, :published)
+      mail = ScheduledPublishMailer.success_email(recipient, edition, edition.status)
+
+      expect(mail.to).to eq([recipient.email])
+
+      mail_subject = I18n.t!("scheduled_publish_mailer.success_email.subject.published",
+                             title: edition.title)
+      expect(mail.subject).to eq(mail_subject)
+    end
+
+    it "errors for an unexpected status" do
```

Comment:
Oops - didn't spot this one earlier. Again, there doesn't seem to be a precedent for this kind of assertion test.